src/imaging/examplepage.md

```diff
@@ -0,0 +1,61 @@
+# Example Page
```

Comment:
Please also link to https://myst-parser.readthedocs.io/en/latest/ or https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#syntax-core, which contains information about extensions to markdown syntax. This is the actual markdown parser we are using.